docs/ldml/tr35-keyboards.md

```diff
@@ -1839,19 +1833,19 @@ The first three transforms above delete various ligatures with a single keypress
 The backspace transform is much like other transforms except in its processing model. If we consider the same transform as in the simple transform example, but as a backspace:
 
 ```xml
-<backspace before="X" from="Y" after="Z" to="B"/>
+<backspace before="X" from="Y" to="B"/>
 ```
 
 This would transform the string:
 
 ```default
-XYZ → XBZ
+XY → XB
 ```
 
 If we mark where the current match position is before and after the transform we see:
 
 ```default
-X Y | Z → X B | Z
+X Y | → X B |
```

Comment:
I looked at this one, I suppose the Z does not add any value and in fact obscures